src/DeleteOldSoftDeletes.php

```diff
@@ -59,10 +59,10 @@ private function deleteOldSoftDeletes()
 
     private function deleteOldSoftDeletesForModel($modelName, $modelConfig, $daysBeforeDeletion)
     {
-        $daysBeforeDeletion = empty($modelConfig['days']) ? $daysBeforeDeletion : $modelConfig['days'];
+        $daysBeforeDeletion = $modelConfig['days'] ?? $daysBeforeDeletion;
 
         $affectedRows = $modelName::onlyTrashed()
-            ->where('deleted_at', '<', Carbon::today()->subDays($daysBeforeDeletion))
+            ->where('deleted_at', '<', (new DateTime)->sub(new DateInterval("P{$daysBeforeDeletion}D"))->format('Y-m-d H:i:s'))
```

Comment:
I'm curious; why the decision to drop Carbon?

Reply:
One less package to require and it was easy to translate into vanilla PHP.

Reply:
👍🏻